doc/api/errors.markdown

```diff
@@ -65,15 +65,15 @@ at which this error was instantiated. An example stacktrace follows:
 The first line is formatted as `<error class name>: <error message>`, and it is followed
 by a series of stack frames (each line beginning with "at "). Each frame describes
 a call site in the program that lead to the error being generated. V8 attempts to
-display a name for each function (by variable name, function name, or object 
```

Comment:
it's amusingly ironic that the errors documentation is so full of these trailing-whitespace _errors_